Data/ByteString/Streaming/Internal.hs

```diff
@@ -333,14 +339,14 @@ unpackBytes bss = dematerialize bss
 
   unpackAppendBytesStrict :: S.ByteString -> Stream (Of Word8) m r -> Stream (Of Word8) m r
   unpackAppendBytesStrict (S.PS fp off len) xs =
-   inlinePerformIO $ withForeignPtr fp $ \base -> do
-        loop (base `plusPtr` (off-1)) (base `plusPtr` (off-1+len)) xs
+    inlinePerformIO $ withForeignPtr fp $ \base ->
```

Comment:
The `bytesting` library is about to remove the long deprecated `inlinePerformIO` as of `0.11` you should use either `accursedUnutterablePerformIO` or (better yet) `unsafeDupablePerformIO` (provided GHC 7.0 is not supported, the latter is only available with GHC 7.2 and up).  The release changelog PR in `bytestring`.

Reply:
[See] (https://github.com/haskell/bytestring/commit/444c475afc34d78d89bae7449c9109f6ee617cae)
By the way it seems this library uses both of the legacy functions, but given that GHC 7.10 is the oldest supported (or at least oldest tested) swapping both for `unsafeDupablePerformIO` seems right to me.

Cc: @bodigrim